src/Update/Nix/FetchGit/Utils.hs

```diff
@@ -73,14 +77,10 @@ matchAttr t = \case
   NamedVar _ _ -> Nothing
   Inherit _ _  -> Nothing
 
-traverse2 :: (Traversable t, Traversable s, Applicative f)
-          => (a -> f b) -> t (s a) -> f (t (s b))
-traverse2 f = fmap getCompose . traverse f . Compose
-
-sequenceA2 :: (Traversable t, Traversable s, Applicative f)
-            => t (s (f a)) -> f (t (s a))
-sequenceA2 = fmap getCompose . sequenceA . Compose
-
-mapConcurrently2 :: (Traversable t, Traversable s) => (a -> IO b) -> t (s a) -> IO (t (s b))
-mapConcurrently2 f = fmap getCompose . mapConcurrently f . Compose
-
+-- Takes an ISO 8601 date and returns just the day portion.
+parseISO8601DateToDay :: Text -> Either Warning Day
```

Comment:
This function probably exists already in the [iso8601-time-0.1.4](https://hackage.haskell.org/package/iso8601-time-0.1.4)